packages/webdriverio/src/commands/element/dragAndDrop.js

```diff
@@ -26,7 +26,9 @@ import { getElementRect, getScrollPosition } from '../../utils'
 
 const ACTION_BUTTON = 0
 
-export default async function dragAndDrop (target, { duration = 100 } = {}) {
+const sleep = (time = 0) => new Promise((resolve) => setTimeout(resolve, time))
+
+export default async function dragAndDrop(target, { duration = 0 } = {}) {
```

Comment:
Is there a reason we should change the default duration?

Reply:
@christian-bromann Actually, if no duration parameter were passed, as per the change, there would be a default delay of 100ms for all dragAndDrop on JsonWire as we use sleep there. That was the only reason why I changed it to 0.

Reply:
@christian-bromann Understood the issue. I should have kept it similar to the format to ensure that the behaviour remains the same across both the protocols and to emulate an actual user action.

Will make this change now.

Reply:
Done.